app/models/manageiq/providers/amazon/cloud_manager/metrics_capture.rb

```diff
@@ -124,11 +130,13 @@ def perf_capture_data_amazon(cloud_watch, start_time, end_time)
 
     counters_by_id              = {target.ems_ref => VIM_STYLE_COUNTERS}
     counter_values_by_id_and_ts = {target.ems_ref => counter_values_by_ts}
+
     return counters_by_id, counter_values_by_id_and_ts
   end
 
   def counter_values_by_timestamp(metrics_by_counter_name)
     counter_values_by_ts = {}
+
```

Comment:
@AlexanderZagaynov I don't disagree with these extra newlines (well not all of them at least) but they're completely unrelated to the rest of this PR.

Can you remove any unrelated whitespace changes?

Reply:
done